x-pack/legacy/plugins/ml/public/data_frame/pages/data_frame_new_pivot/components/step_define/pivot_preview.tsx

```diff
@@ -25,7 +25,7 @@ import {
   ColumnType,
   MlInMemoryTable,
   SORT_DIRECTION,
-} from '../../../../../../common/types/eui/in_memory_table';
+} from '../../../../../components/ml_in_memory_table';
```

Comment:
As above, this table fails to load for me, due to the missing `pageIndex` prop.

Reply:
Fixed in d981ba0a0f7c62b781cc099bff9ef1c2fa58d7c7